js/components/conceptset/InputTypes/ConceptSetItem.js

```diff
@@ -5,9 +5,9 @@ define(['knockout', 'conceptpicker/InputTypes/Concept'], function (ko, Concept)
 		var self = this;
 		
 		self.concept = data.concept && new Concept(data.concept);
-		self.isExcluded = ko.observable(data.isExcluded || false);
-		self.includeDescendants = ko.observable(data.includeDescendants || false);
-		self.includeMapped = ko.observable(data.includeMapped || false);
+		self.isExcluded = ko.observable(data.isExcluded && data.isExcluded() || false);
+		self.includeDescendants = ko.observable(data.includeDescendants && data.includeDescendants() || false);
+		self.includeMapped = ko.observable(data.includeMapped && data.includeMapped() || false);
```

Comment:
Don't do this, data should never come in as an observable to the constructor.

If you want to make a 'copy' and re-observe it, take the origional, do a ko.toJS() on the object, then re-instantiate.

ie:

```
var origional; // this is the existing thing to clone

var clone = new ConceptSetItem(ko.toJS(origional))
```

```

Reply:
Ok, reverted